Samples/EmbeddedAuthWithSDKs/EmbeddedAuth/View Controllers/ProfileTableViewController.swift

```diff
@@ -117,7 +133,7 @@ class ProfileTableViewController: UITableViewController {
         DispatchQueue.main.async {
             let alert = UIAlertController(title: nil, message: error.localizedDescription, preferredStyle: .alert)
             alert.addAction(.init(title: "OK", style: .default))
-            self.show(alert, sender: nil)
+            self.present(alert, animated: true)
```

Comment:
What's the difference between show and present?

Reply:
"Show" means "Display this view controller in the detail area of the screen".  So in a split-view application, it will show it in the right-hand side.  In a navigation controller app (which this is), it will attempt to push it onto the navigation stack, which doesn't work (or make sense) with an alert.  

So this was just a bone-headed bug which I'd never really encountered before, and needed to update.